### PR TITLE
[FR] Update `Judgement/osu!`

### PR DESCRIPTION
**Pull request by DoryanR**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Judgement/osu!/fr.md [hunk 1]
```diff
@@ -1,62 +1,90 @@
...
+Ils sont généralement désignés par la valeur de leur score (sauf pour les miss), c'est-à-dire qu'un GREAT est généralement appelé un "300" et ainsi de suite.
+
+| Image | Nom | [Valeur de hit](/wiki/Gameplay/Score/ScoreV1/osu!) | [Précision](/wiki/Gameplay/Accuracy#osu!) | Erreur de hit maximum (ms) |
+| :-: | :-: | --: | --: | :-- |
+| ![](/wiki/shared/judgement/osu!/hit300.png) | GREAT | 300 | 100% | `80 - 6 × OD` |
+| ![](/wiki/shared/judgement/osu!/hit100.png) | OK | 100 | 33.33% | `140 - 8 × OD` |
+| ![](/wiki/shared/judgement/osu!/hit50.png) | MEH | 50 | 16.67% | `200 - 10 × OD` |
+| ![](/wiki/shared/judgement/osu!/hit0.png) | MISS | 0 | 0% | `400` |
+
+La fenêtre de hit dépend de la [difficulté globale (OD)](/wiki/Beatmap/Overall_difficulty) de la beatmap. Un hit est considéré comme étant à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximale`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre de hit.
```
Comment (Biokax) [inline wiki/Gameplay/Judgement/osu!/fr.md:16]: comme pour la pr sur judgement/taiko, voir pour rajouter "overall difficulty" pour rester cohérent avec la page associée

File: wiki/Gameplay/Judgement/osu!/fr.md [hunk 2]
```diff
@@ -1,62 +1,90 @@
...
-  - Le cercle n'est pas touché et la hit window du 50 est passée.
+### Hit circle
+
+Un [hit circle](/wiki/Gameplay/Hit_object/Hit_circle) est jugé avec une mention GREAT, OK, MEH, ou MISS en fonction de la précision du hit. Hit un hit circle avant la fenêtre MISS n'a pas d'effet (autre que de causer un [notelock](/wiki/Gameplay/Judgement/Notelock)), et ne pas hit un hit circle causera un MISS après que la fenêtre MEH soit passée.
 
 ### Sliders
 
-Les sliders sont composés de plusieurs parties : la début du slider, la fin du slider, les ticks du slider et les répétitions du slider. Le slider dans son ensemble est jugé en fonction du nombre de ses parties que le joueur a touché, comme indiqué ci-dessous :
+Les [sliders](/wiki/Gameplay/Hit_object/Slider) sont constitués de plusieurs parties : la [slider head](/wiki/Gameplay/Hit_object/Slider/Sliderhead), la [slider tail](/wiki/Gameplay/Hit_object/Slider/Slidertail), les [sliders ticks](/wiki/Gameplay/Hit_object/Slider/Slider_tick), et les [sliders repeats](/wiki/Gameplay/Hit_object/Slider/Reverse_slider). Le slider dans son ensemble est jugé en fonction du nombre de parties que le joueur a hit, comme indiqué ci-dessous :
```
Comment (Biokax) [inline wiki/Gameplay/Judgement/osu!/fr.md:40]:
```suggestion
Les [sliders](/wiki/Gameplay/Hit_object/Slider) sont constitués de plusieurs parties : le [slider head](/wiki/Gameplay/Hit_object/Slider/Sliderhead), le [slider tail](/wiki/Gameplay/Hit_object/Slider/Slidertail), les [sliders ticks](/wiki/Gameplay/Hit_object/Slider/Slider_tick), et les [sliders repeats](/wiki/Gameplay/Hit_object/Slider/Reverse_slider). Le slider dans son ensemble est jugé en fonction du nombre de parties que le joueur a hit, comme indiqué ci-dessous :
```

File: wiki/Gameplay/Judgement/osu!/fr.md [hunk 3]
```diff
@@ -1,62 +1,90 @@
...
+| MISS | 0% |
 
-- 300 : Le joueur a réussi à toucher toutes les parties du slider.
-- 100 : Le joueur a au moins touché la moitié des parties du slider.
-- 50 : Le joueur a réussi à toucher au moins une des parties du slider.
-- Miss : Le joueur n'a touché aucune des parties du slider.
-- Si le [ScoreV2](/wiki/Gameplay/Game_modifier/ScoreV2) est actif, la précision avec laquelle le joueur touche le début du slider est également prise en compte dans le jugement de l'ensemble du slider :
-  - Obtenir un 300 sur un slider nécessite d'obtenir un jugement de 300 sur le début du slider.
-  - Obtenir un 100 sur un slider nécessite d'obtenir un jugement de 100 ou mieux sur le début de slider.
+Il suffit que la slider head soit touchée dans la fenêtre de hit de la MEH pour que le hit soit réussi. Cependant, si [ScoreV2](/wiki/Gameplay/Game_modifier/ScoreV2) est actif, la précision de la slider head est également prise en compte, le jugement reçu pour l'ensemble du slider étant limité au maximum à celui reçu pour la slider head.
```
Comment (Biokax) [inline wiki/Gameplay/Judgement/osu!/fr.md:49]:
```suggestion
Il suffit que le slider head soit touché dans la fenêtre de hit de la MEH pour que le hit soit réussi. Cependant, si [ScoreV2](/wiki/Gameplay/Game_modifier/ScoreV2) est actif, la précision du slider head est également prise en compte, le jugement reçu pour l'ensemble du slider étant limité au maximum à celui reçu pour le slider head.
```

File: wiki/Gameplay/Judgement/osu!/fr.md [hunk 4]
```diff
@@ -1,62 +1,90 @@
...
 
-Les parties du slider ont également une influence supplémentaire sur le [combo](/wiki/Gameplay/Combo_(score_multiplier)) actuel :
+Il existe d'autres particularités concernant l'influence des sliders dans les [combos](/wiki/Gameplay/Combo_(score_multiplier)) :
 
-- Toucher le début du slider trop tôt n'entraîne pas un miss, mais remet le combo à 0.
-- Rater un tick du slider ou une répétition n'entraîne pas un miss, mais remet le combo à 0.
-- Rater la fin du slider n'entraîne pas un miss, mais n'incrémentera pas le combo.
+- Hit la slider head trop tôt (avant la fenêtre MEH), miss un slider tick, ou un repeat n'entraîne pas un MISS, mais causera un [combo break](/wiki/Gameplay/Judgement/Combobreak). Les autres parties du slider peuvent toujours être hit si une touche est maintenue enfoncée. C'est ce qu'on appelle familièrement un [slider break](/wiki/Gameplay/Judgement/Slider_break).
```
Comment (Biokax) [inline wiki/Gameplay/Judgement/osu!/fr.md:53]:
```suggestion
- Hit le slider head trop tôt (avant la fenêtre MEH), miss un slider tick, ou un repeat n'entraîne pas un MISS, mais causera un [combo break](/wiki/Gameplay/Judgement/Combobreak). Les autres parties du slider peuvent toujours être hit si une touche est maintenue enfoncée. C'est ce qu'on appelle familièrement un [slider break](/wiki/Gameplay/Judgement/Slider_break).
```

File: wiki/Gameplay/Judgement/osu!/fr.md [hunk 5]
```diff
@@ -1,62 +1,90 @@
...
+Si un spinner est très court, le nombre de spins requis peut être calculé comme étant égal à 0, et le spinner se terminera donc toujours par un GREAT.
 
 ## Histoire
 
-L'algorithme de jugement du spinner a été modifié de manière significative dans le build [20190510.2 Cutting Edge release](https://osu.ppy.sh/home/changelog/cuttingedge/20190510.2). Avant ce changement, il était beaucoup plus difficile d'obtenir des jugements sans erreur, car la différence entre un 50 et un 100, ainsi qu'entre un 100 et un 300, était égale à la moitié d'un tour. Certaines beatmaps comportaient même des spinners qui ne pouvaient jamais être jugés avec un 300, car ils étaient trop courts pour être complétés entièrement.
+L'algorithme de jugement du spinner a été considérablement modifié dans la [mise à jour Stable 20190513.2](https://osu.ppy.sh/home/changelog/stable40/20190513.2). Les différences sont les suivantes :
+
+- La différence entre OK et MEH, ainsi qu'entre MEH et GREAT, était égale à un demi spin, ce qui rendait les jugements non-MISS beaucoup plus difficiles à réaliser.
+- Le fait que les spinners soient trop courts peut les avoir empêchés d'être complètement clear.
```
Comment (Biokax) [inline wiki/Gameplay/Judgement/osu!/fr.md:81]:
```suggestion
- Le fait que les spinners soient trop courts a pu les empêcher d'être complètement clear.
```

**Review by Biokax — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Judgement/osu!/fr.md (hunk 1 above)
Comment (DoryanR) [inline wiki/Gameplay/Judgement/osu!/fr.md:16]:
```suggestion
La fenêtre de hit dépend de la [difficulté globale (OD)](/wiki/Beatmap/Overall_difficulty) (overall difficulty) de la beatmap. Un hit est considéré comme étant à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximale`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre de hit.
```

File: wiki/Gameplay/Judgement/osu!/fr.md [hunk 6]
```diff
@@ -1,62 +1,90 @@
...
+Ils sont généralement désignés par la valeur de leur score (sauf pour les miss), c'est-à-dire qu'un GREAT est généralement appelé un "300" et ainsi de suite.
+
+| Image | Nom | [Valeur de hit](/wiki/Gameplay/Score/ScoreV1/osu!) | [Précision](/wiki/Gameplay/Accuracy#osu!) | Erreur de hit maximum (ms) |
+| :-: | :-: | --: | --: | :-- |
+| ![](/wiki/shared/judgement/osu!/hit300.png) | GREAT | 300 | 100% | `80 - 6 × OD` |
+| ![](/wiki/shared/judgement/osu!/hit100.png) | OK | 100 | 33.33% | `140 - 8 × OD` |
+| ![](/wiki/shared/judgement/osu!/hit50.png) | MEH | 50 | 16.67% | `200 - 10 × OD` |
+| ![](/wiki/shared/judgement/osu!/hit0.png) | MISS | 0 | 0% | `400` |
+
+La fenêtre de hit dépend de la [difficulté globale (OD)](/wiki/Beatmap/Overall_difficulty) (overall difficulty) de la beatmap. Un hit est considéré comme étant à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximale`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre de hit.
```
Comment (DoryanR) [inline wiki/Gameplay/Judgement/osu!/fr.md:16]:
```suggestion
La fenêtre de hit dépend de la [overall difficulty (OD)](/wiki/Beatmap/Overall_difficulty) de la beatmap. Un hit est considéré comme étant à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximale`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre de hit.
```

File: wiki/Gameplay/Judgement/osu!/fr.md [hunk 7]
```diff
@@ -1,62 +1,90 @@
...
+Ils sont généralement désignés par la valeur de leur score (sauf pour les miss), c'est-à-dire qu'un GREAT est généralement appelé un "300" et ainsi de suite.
+
+| Image | Nom | [Valeur de hit](/wiki/Gameplay/Score/ScoreV1/osu!) | [Précision](/wiki/Gameplay/Accuracy#osu!) | Erreur de hit maximum (ms) |
+| :-: | :-: | --: | --: | :-- |
+| ![](/wiki/shared/judgement/osu!/hit300.png) | GREAT | 300 | 100% | `80 - 6 × OD` |
+| ![](/wiki/shared/judgement/osu!/hit100.png) | OK | 100 | 33.33% | `140 - 8 × OD` |
+| ![](/wiki/shared/judgement/osu!/hit50.png) | MEH | 50 | 16.67% | `200 - 10 × OD` |
+| ![](/wiki/shared/judgement/osu!/hit0.png) | MISS | 0 | 0% | `400` |
+
+La fenêtre de hit dépend de la [overall difficulty (OD)](/wiki/Beatmap/Overall_difficulty) de la beatmap. Un hit est considéré comme étant à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximale`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre de hit.
```
Comment (DoryanR) [inline wiki/Gameplay/Judgement/osu!/fr.md:16]:
```suggestion
La fenêtre de hit dépend de l'[overall difficulty (OD)](/wiki/Beatmap/Overall_difficulty) de la beatmap. Un hit est considéré comme étant à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximale`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre de hit.
```

File: wiki/Gameplay/Judgement/osu!/fr.md [hunk 8]
```diff
@@ -1,62 +1,90 @@
...
+Ils sont généralement désignés par la valeur de leur score (sauf pour les miss), c'est-à-dire qu'un GREAT est généralement appelé un "300" et ainsi de suite.
+
+| Image | Nom | [Valeur de hit](/wiki/Gameplay/Score/ScoreV1/osu!) | [Précision](/wiki/Gameplay/Accuracy#osu!) | Erreur de hit maximum (ms) |
+| :-: | :-: | --: | --: | :-- |
+| ![](/wiki/shared/judgement/osu!/hit300.png) | GREAT | 300 | 100% | `80 - 6 × OD` |
+| ![](/wiki/shared/judgement/osu!/hit100.png) | OK | 100 | 33.33% | `140 - 8 × OD` |
+| ![](/wiki/shared/judgement/osu!/hit50.png) | MEH | 50 | 16.67% | `200 - 10 × OD` |
+| ![](/wiki/shared/judgement/osu!/hit0.png) | MISS | 0 | 0% | `400` |
+
+La fenêtre de hit dépend de l'[overall difficulty (OD)](/wiki/Beatmap/Overall_difficulty) de la beatmap. Un hit est considéré comme étant à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximale`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre de hit.
```
Comment (DoryanR) [inline wiki/Gameplay/Judgement/osu!/fr.md:16]:
```suggestion
La fenêtre de hit dépend de [l'overall difficulty (OD)](/wiki/Beatmap/Overall_difficulty) de la beatmap. Un hit est considéré comme étant à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximale`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre de hit.
```

**Review by cl8n — APPROVED** [review]

**Pull request merged**